jenkins_pipelines/environments/common/pipeline-build-validation-aws.groovy

```diff
@@ -608,7 +633,11 @@ def getNodesHandler() {
     String[] moduleList = modules.split("\n")
     moduleList.each { lane ->
         def instanceList = lane.tokenize(".")
-        if (instanceList[1].contains('minion') || instanceList[1].contains('client')) {
```

Comment:
What about using:
`def nodeName = lane.tokenize(".")[1]`

And then you can replace `instanceList[1]` by `nodeName` ?